app/javascript/react/screens/App/Overview/components/ModalWizard/ModalWizard.js

```diff
@@ -14,10 +16,23 @@ const ModalWizard = props => {
     activeStep,
     numSteps,
     goToStep,
-    children
+    children,
+    formContainer
   } = props;
   const onFirstStep = activeStepIndex === 0;
   const onFinalStep = activeStepIndex === numSteps - 1;
+
+  const formHasErrors = () =>
+    find(formContainer, form => form.syncErrors) !== undefined;
+
+  const disableNextStep = formHasErrors();
```

Comment:
This function definition with only one immediate call could be simplified to a variable assignment:

```
const formHasErrors = find(formContainer, form => form.syncErrors) !== undefined;
const disableNextStep = formHasErrors;
```

Reply:
Addressed in d3da630